README.md

```diff
@@ -309,17 +309,78 @@ func main() {
 }
 ```
 
+### Using Paging
+
+This library also offers paging functionality. Collections such as calls and messages have `ListXxx` and `StreamXxx`
+functions that page under the hood. With both list and stream, you can specify the number of records you want to receive (limit) and the maximum size you want each page fetch to be (pageSize). The library will then handle the task for you.
+
+`List` eagerly fetches all records and returns them as a list, whereas `Stream` streams the records and lazily retrieves
+the pages as you iterate over the collection. You can also page manually using the `PageXxx` function in each of the apis.
+
+```go
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/twilio/twilio-go"
+	openapi "github.com/twilio/twilio-go/rest/api/v2010"
+	"os"
+)
+
+func main() {
+	accountSid := os.Getenv("TWILIO_ACCOUNT_SID")
+	authToken := os.Getenv("TWILIO_AUTH_TOKEN")
+	from := os.Getenv("TWILIO_FROM_PHONE_NUMBER")
+	
+	client := twilio.NewRestClient(accountSid, authToken)
+
+	params := &openapi.ListMessageParams{}
+	params.SetFrom(from)
+	params.SetPageSize(20)
+	limit := 100
+	
+	resp, err := client.ApiV2010.ListMessage(params, &limit)
+	for record := range resp {
+		for i := range resp[record].Messages {
+			fmt.Println("From: ", *resp[record].Messages[i].Body)
+		}
+	}
+
+	resp, err = client.ApiV2010.PageMessage(params, "pagetoken", "1")
+	if err != nil {
+		fmt.Println(err)
+	} else {
+		fmt.Println(resp.Messages[0].Body)
+	}
```

Comment:
Think it would be better to move this to its own example since filling in the page token is based on the response from the first call the `PageMessage`.